src/server/utils/sendSegmentEvent.js

```diff
@@ -11,7 +12,8 @@ const getTraits = (userIds) => {
       createdAt: r.row('createdAt').default(0),
       email: r.row('email').default(''),
       id: r.row('id').default(''),
-      name: r.row('preferredName').default('')
+      name: r.row('preferredName').default(''),
+      parabolId: r.row('id').default('') // passed as a distinct trait name for HubSpot
```

Comment:
Silly question, why can't you use the already existing id field?

Reply:
It's actually interesting, or lame, depending on your love for Segment and HubSpot.

Segment will publish its traits to HubSpot as HubSpot Contact Properties. However, there are some reserved words and `id` is one of them. So `parabolId` it is! (Which will actually be xformed to snake-case as `parabol_id` by the Segment HubSpot integration).

It's all duct-tape and bailing wire.